docs/source/release_notes.rst

```diff
@@ -1,6 +1,18 @@
 Release Notes
 -------------
 **Future Releases**
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. warning::
+
+    **Breaking Changes**
+
+
+**v0.33.0 Sep. 10, 2021**
```

Comment:
Hehe mild nitpick about date?

Reply:
lol that's not a nit, it's just a straight up mistake.  Thanks for the catch.